src/Libraries/Data/List/Quantifiers/Extra.idr

```diff
@@ -14,10 +14,12 @@ lookup v (px :: pxs)
       No _ => lookup v pxs
       Yes Refl => Just px
 
+-- TODO: delete this function once we release the next compiler version
+-- (after 0.7.0) as it has been replicated in base's Data.List.Quantifiers
```

Comment:
Could you please change the wording to something like "delete this function after 0.7.1 is released" because wording "next release after 0.7.0" proved to be really confusing (well, at least for me, doing cleanups). I think wording mentioning `0.7.1` is okay even if next release would be `0.8.0`.

Reply:
done. All good?